cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -107,6 +109,40 @@ def test_errored_course_global_staff(self):
             courses_list_by_groups, __ = _accessible_courses_list_from_groups(self.request)
             self.assertEqual(courses_list_by_groups, [])
 
+    @ddt.data(
+        (ModuleStoreEnum.Type.split, 5),
+        (ModuleStoreEnum.Type.mongo, 3)
+    )
+    @ddt.unpack
+    def test_staff_course_listing(self, default_store, mongo_calls):
+        """
+        Create courses and verify they take certain amount of mongo calls to call get_courses_accessible_to_user.
+        Also verify that fetch accessible courses list for staff user returns CourseSummary instances.
+        """
+
+        # Assign & verify staff role to the user
+        GlobalStaff().add_users(self.user)
+        self.assertTrue(GlobalStaff().has_user(self.user))
+
+        # Create few courses
+        for num in xrange(TOTAL_COURSES_COUNT):
```

Comment:
@adampalay Not sure I totally understand why we should change `num` to `__`. Can you please explain?